napari_aicssegmentation/controller/workflow_steps_controller.py

```diff
@@ -34,24 +34,25 @@ def close_workflow(self):
         self.router.workflow_selection()
 
     def run_all(self, parameter_inputs: List[Dict[str, List]]):
-        """ 
-        parameter_inputs is a list of dictionaries, one dictionary (or None if no parameters) 
-        for each step. Each dictionary has the same shape as a WorkflowStep.parameter_defaults
-        dictionary, but with the parameter values obtained from the UI instead of default values.
         """
+        Run all steps in the active workflow.
 
-        workflow = self.model.active_workflow
+        parameter_inputs List[Dict]/Dict: Each dictionary has the same shape as a WorkflowStep.parameter_defaults
```

Comment:
Does `List[Dict]/Dict` here mean `parameter_inputs` can be a list of dictionaries or it can be a single dictionary? Because it would always be a list of dictionaries (or a list containing a single dictionary if there is only 1 step).

Actually, if we want to be super accurate it would be a list containing dictionaries and/or Nones (for workflow steps with no params) haha